src/test/java/teammates/test/cases/BaseTestCaseWithDatastoreAccess.java

```diff
@@ -199,8 +200,10 @@ protected CourseAttributes getCourse(CourseAttributes course) {
     }
 
     private void equalizeIrrelevantData(CourseAttributes expected, CourseAttributes actual) {
-        // Ignore time field as it is stamped at the time of creation in testing
+        // ignore time field as it is stamped at the time of creation in testing
         expected.createdAt = actual.createdAt;
+        // sanitize fields
+        expected.sanitizeForSaving();
```

Comment:
Why there are things that are irrelevant to this PR?
Although we need sanitise field here.